Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1816,6 +1712,28 @@ PhysicalParticleContainer::AddPlasmaFlux (PlasmaInjector const& plasma_injector,
         {
             const IntVect iv = IntVect(AMREX_D_DECL(i, j, k));
             const auto index = overlap_box.index(iv);
+
+            Real scale_fac = compute_scale_fac_area(dx, num_ppc_real, flux_normal_axis);
+
+            auto lo = getCellCoords(overlap_corner, dx, {0._rt, 0._rt, 0._rt}, iv);
+            auto hi = getCellCoords(overlap_corner, dx, {1._rt, 1._rt, 1._rt}, iv);
+
+            if (flux_pos->overlapsWith(lo, hi))
+            {
+                int r;
+                if (fine_overlap_box.ok() && fine_overlap_box.contains(iv)) {
+                    r = compute_area_weights(lrrfac, flux_normal_axis);
+                } else {
+                    r = 1;
+                }
+                scale_fac /= r;
+            }
+#if defined(WARPX_DIM_XZ) || defined(WARPX_DIM_RZ)
+            amrex::ignore_unused(k);
+#elif defined(WARPX_DIM_1D_Z)
+            amrex::ignore_unused(j,k);
```

Comment:
The macros can be removed and only have this one line `amrex::ignore_unused(j,k);'